raiden-ts/src/transport/epics/messages.ts

```diff
@@ -86,6 +89,10 @@ export function matrixMessageSendEpic(
         { log, latest$, config$ },
         true, // allowRtc
       ).pipe(
+        retryWaitWhile(
```

Comment:
`waitMemberAndSend$` already has a retry mechanism embedded. I think it's better to extend it than to add another retry layer here.

Reply:
Good point! Fixed